lib/build.py

```diff
@@ -381,6 +396,7 @@ def init(self, parent):
          self.image = self.parent.image
          self.image_alias = self.parent.image_alias
          self.image_i = self.parent.image_i
+         self.argfrom = self.parent.argfrom
```

Comment:
Is it possible to make the argfrom an attribute of the Dockerfile? Does that make it a global?